docs/01-app/01-getting-started/05-css.mdx

```diff
@@ -103,45 +103,22 @@ export default function RootLayout({ children }) {
 
 ### Installing Tailwind
 
-To start using Tailwind, install the Tailwind CSS packages and run the `init` command to generate both the `tailwind.config.js` and `postcss.config.js` files:
+To start using Tailwind, install the necessary Tailwind CSS packages:
 
 ```bash filename="Terminal"
-npm install -D tailwindcss postcss autoprefixer
-npx tailwindcss init -p
```

Comment:
I wonder if it should still use the init, or is that not recommended anymore?

Reply:
@leerob It's not recommended anymore according to the latest tailwindcss v4 docs.
https://tailwindcss.com/docs/installation/framework-guides/nextjs